docs/api/monitoring/monitoring-adapter.md

```diff
@@ -6,6 +6,15 @@ description: This document describes the interface that a monitoring adapter sho
 
 ## Monitoring Adapter API
 
+:::info TypeScript Support
+
+A monitoring adapter object has an associated TypeScript type:
+
+- Type Name - `MonitoringAdapter`
+- Package - `@resolve-js/core`
+
+:::
+
 A monitoring adapter implementation should expose the following interface:
 
```

Comment:
Lines 33 and 43 (below):
`| Argument Name | Type               | Descriptions                           |`

I see only one list item in the tables. Should "Descriptions" be "Description"?

Reply:
> Please change "Descriptions" to "Description".

Done

> What do you mean by "a monitoring"?

I dropped the word "adapter". Changed to "The code below demonstrates how to implement a monitoring adapter that uses [Prometheus](https://prometheus.io/) to publish metrics."

Reply:
Sounds good. Thank you.